### ISSUE TITLE
Suggesstion about session close

### ISSUE THREAD

**Issue by BetterYan** [nfmcclure/tensorflow_cookbook#107]
Could you please add line "sess.close()" for each examples?
It will mislead new comers like me. :-)

**Comment by nfmcclure** [comment]
Yes of course!  You should see these start to appear in the next few months. Thanks for the suggestion and sorry for the late response.
